frontend/src/components/AppLogo/AppLogo.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Spoon from '../../assets/Spoon.png';
+
+const AppLogo = () => {
+  return (
+    <img src={Spoon} alt="spoon logo" style={{ width: '50px', }} />
+  )
+}
+
+export default AppLogo;
```

Comment:
I think it's ok to do, but I don't know how necessary it is to separate out the logo img into a separate component as it looks like it's set up to only be in the NavBar component.  IF you think this will be used other places as an image you can keep it as a separate component, but delete the appLogo.styles file since it is empty and you're using inline styles here.